properties.schema

```diff
@@ -419,84 +429,84 @@
             "type": "string",
             "inputType": "ColourPicker",
             "default": "#36CDE8",
-            "help": "Notify background colour"
+            "help": "Sets the Notify (pop up) colour."
```

Comment:
I'm not sure this help text really adds anything as the title is sufficiently clear by itself.